### PR TITLE
display all cover images with 1.9 aspect ratio

### PR DESCRIPTION
**Pull request by st-jay**
## Connected Issues
- Issue #18 

## Summary of goal
Display all cover images in the blog listing page with a uniform aspect ratio

## Summary of solution
- Use `object-fit: cover` for all images so that they resize to fill the container ( which has an aspect ratio 1.9 ) without stretching.

### Remaining TODOs
- [ ] ~Remaining change...~

## Results
Screenshot 2022-03-07 at 11 35 25 AM

#### Tested on all primary browsers for:
- Chrome
  - [X] Desktop
  - [X] Mobile
  - [X] Tablet
- Safari
  - [X] Desktop
  - [X] Mobile
  - [X] Tablet
- Firefox
  - [X] Desktop
  - [X] Mobile
  - [X] Tablet
- (Optional) Tested on Safari 12 (Physical or emulator)
  - [X] iPad
  - [X] iPhone
- (Optional) Tested on physical mobile and tablet device for:
  - [ ] ~Android~
  - [ ] ~iOS (including iPadOS)~

#### Tested analytics events
- [ ] ~All pre-existing events are working~
- Newly added analytic events
  - [ ] ~description of when the event would occur~

### CODE REVIEW

File: src/styles/blog.css [hunk 1]
```diff
@@ -72,18 +72,20 @@
 }
 
 .blog-card__image-container {
-  display: flex;
-  justify-content: center;
-  align-items: center;
   overflow: hidden;
   width: 360px;
-  height: 160px;
+  padding-top: 52.5%;
```
Comment (nkshah2) [inline src/styles/blog.css:77]: What is this for?

File: src/styles/blog.css [hunk 2]
```diff
@@ -72,18 +72,20 @@
...
   border-radius: 4px;
+  position: relative;
 }
 
 .blog-card__image {
-  min-height: 100%;
   width: 100%;
```
Comment (nkshah2) [inline src/styles/blog.css:83]: lets have the image `object-fit` mode set to `cover`. This way the images will all have the same render size and we can fix the ones that zoom in over time

**Review by nkshah2 — CHANGES_REQUESTED** [review]

File: src/styles/blog.css (hunk 1 above)
Comment (st-jay) [reply src/styles/blog.css:77]: That's the aspect ratio fix, as per [this](https://www.w3schools.com/howto/howto_css_aspect_ratio.asp) article. We want our aspect ratio to be 40:21 ( which is the image's source aspect ratio ), so we set the `padding-top` to `(21/40 * 100)%` which is `52.5%`. It works on chrome, safari and firefox browsers.

Comment (st-jay) [reply src/styles/blog.css]: Can be ignored as we do not need this method anymore

File: src/styles/blog.css (hunk 2 above)
Comment (st-jay) [reply src/styles/blog.css:83]: updated

Comment (nkshah2) [reply src/styles/blog.css:83]: The aspect ratio on mobile is wrong

**Review by nkshah2 — CHANGES_REQUESTED** [review]

File: src/styles/blog.css (hunk 2 above)
Comment (st-jay) [reply src/styles/blog.css:83]: updated

**Pull request merged**